src/content/docs/query-your-data/nrql-new-relic-query-language/get-started/query-time-range.mdx

```diff
@@ -0,0 +1,2701 @@
+          <span style="font-family:monospace">Time</span>, with the time value defaulting to midnight at the start of
+          the given date.
+
+          The dashes between date elements are optional (but recommended), so `'20231218'` is equivalent to `'2023-12-18'`.
+        </td>
+      </tr>
+      <tr>
+        <td><span style="font-family:monospace">TimeSeparator</span></td>
+        <td>
+          Separates the date and time in a full DateTime string. Valid separating values are the capital letter `T` or a single space ` `.
+        </td>
+      </tr>
+      <tr>
+        <td><span style="font-family:monospace">Time</span></td>
+        <td>
+          <div style="font-family:monospace">Hour:Minute[[:Second].FractionalSecond]</div>
+
+          <div>
+          * <span style="font-family:monospace">Hour</span> - (*required*) 1 or 2 digit hour value, 00-23.
+          2 digits is recommended, per the ISO 8601 specification.
+          * <span style="font-family:monospace">Minute</span> - (*required*) 2 digit minute value, 00-59.
+          * <span style="font-family:monospace">:Second</span> - (*optional*) 2 digit seconds value, 00-59.
+          * <span style="font-family:monospace">[.FractionalSecond]</span> - (*optional*) 1 to
+          3 digit fractional second (milliseconds) value, 000-999. Only available when a <span
+          style="font-family:monospace">:Second</span> value is included.
+          </div>
+
+          A DateTime string can include a <span style="font-family:monospace">Time</span>
+          without a <span style="font-family:monospace">Date</span>, with the date value defaulting
+          to the current day. When a time is given without a date in the `SINCE` clause,
+          if the time is in the future, the previous day will be used for the date.
+
+          When a <span style="font-family:monospace">Time</span> follows a `T` <span
+          style="font-family:monospace">TimeSeparator</span>, the colons are optional (but
+          recommended). If the colons are excluded, the <span
+          style="font-family:monospace">Hour</span> must include 2 digits and <span
+          style="font-family:monospace">AM/PM</span> is not allowed. So,
+          `'20231218T123456.789'` is equivalent to `'2023-12-18T12:34:56.789'`.
+        </td>
+      </tr>
+      <tr>
+        <td><span style="font-family:monospace">TimeZone</span></td>
+        <td>
+          *Optional* - If <span style="font-family:monospace">TimeZone</span> is not
+          included in a DateTime string, the time zone indicated in the query's
+          `WITH TIMEZONE` clause will be used if present, otherwise the UTC timezone will be used.
+
+          The <span style="font-family:monospace">TimeZone</span> value can optionally be
+          preceded by a space. It can be indicated in various formats.
+
+          * Offset - Indicates the timezones hours and minutes relative to UTC.
```

Comment:
timezone's

Reply:
It was underlined in red after I changed it - making me rezlize it should be "timezone hours and minutes"